app/controllers/api/api_controller.rb

```diff
@@ -20,6 +20,15 @@ def index
       render_resource :entrypoint, res
     end
 
+    def product_info
```

Comment:
Won't this be overridden by https://github.com/ManageIQ/manageiq-api/pull/438/files#diff-0ef0779741ad3209b46314a66c353351R89 ?

Reply:
Is this also implying that we already have this endpoint?

Reply:
I should not make PRs when I'm tired :cry: I'll fix this tomorrow :confused: :sleeping:

Reply:
I don't think so, because the one below is private.

Reply:
Yup, my bad ... fixed now.

Reply:
Does this fix it though?  It seems this break API compatibility because the old product_info doesn't match the new one.

Reply:
Oh, you're right, is it okay if I merge the `branding_info` into `product_info`?

Reply:
I'm fine with putting the branding info under `product_info`, but did we somehow have `/product_info` before?
![image](https://user-images.githubusercontent.com/651659/46815742-84da3600-cd49-11e8-8c9e-5032ec57a721.png)

Reply:
No, it's a new thing.